prow/plugins/help/help.go

```diff
@@ -30,18 +30,31 @@ import (
 const pluginName = "help"
 
 var (
-	helpLabel         = "help wanted"
-	helpRe            = regexp.MustCompile(`(?mi)^/help\s*$`)
-	helpRemoveRe      = regexp.MustCompile(`(?mi)^/remove-help\s*$`)
-	helpGuidelinesURL = "https://git.k8s.io/community/contributors/devel/help-wanted.md"
-	helpMsgPruneMatch = "This request has been marked as needing help from a contributor."
-	helpMsg           = `
-This request has been marked as needing help from a contributor.
+	helpLabel                  = "help wanted"
+	goodFirstIssueLabel        = "good first issue"
+	helpRe                     = regexp.MustCompile(`(?mi)^/help\s*$`)
+	helpRemoveRe               = regexp.MustCompile(`(?mi)^/remove-help\s*$`)
+	helpGoodFirstIssueRe       = regexp.MustCompile(`(?mi)^/good-first-issue\s*$`)
+	helpGoodFirstIssueRemoveRe = regexp.MustCompile(`(?mi)^/remove-good-first-issue\s*$`)
+	helpGuidelinesURL          = "https://git.k8s.io/community/contributors/devel/help-wanted.md"
+	helpMsgPruneMatch          = "This request has been marked as needing help from a contributor."
+	helpMsg                    = `
+	This request has been marked as needing help from a contributor.
 
 Please ensure the request meets the requirements listed [here](` + helpGuidelinesURL + `).
 
 If this request no longer meets these requirements, the label can be removed
 by commenting with the ` + "`/remove-help`" + ` command.
+`
+	goodFirstIssueGuidelinesURL = "https://git.k8s.io/community/contributors/devel/help-wanted.md#good-first-issue"
```

Comment:
It is not super obvious to me what this URL is gaining us over just inlining it into `goodFirstIssueMsg`?

Reply:
It provides guidelines/expectations for experienced users who use this label. It's one of the easiest ways to get close to a self-sustaining mini mentorship cycle. Not everyone's comfortable pinging people for questions, or has the benefit of knowing a lot of folks personally.

Reply:
Agree on the value of making the comment reference https://git.k8s.io/community/contributors/devel/help-wanted.md#good-first-issue

What is less clear to me is the need to put this in a variable that is only used once
```
// Less clear on why we're doing this
goodFirstIssueGuidelinesURL := "https://foo"
`[here](` + ugoodFirstIssueGuidelinesURL + `)`

// Rather than
`[here](https://foo)`
```

Reply:
> What is less clear to me is the need to put this in a variable that is only used once

Fixed. Ptal.